src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -1,12 +1,13 @@
 import * as React from 'react';
-import {Animated, StyleSheet, TouchableOpacity, View} from 'react-native';
+import {Animated, ImageBackground, StyleSheet, TouchableOpacity, View} from 'react-native';
 import {useNavigation} from '@react-navigation/core';
 import PropTypes from 'prop-types';
 
 import MemoIcArrowBack from '../../../assets/arrow/Ic_arrow_back';
+import MemoIcArrowBackCircle from '../../../assets/arrow/Ic_arrow_back_circle';
 import TopicDefaultIcon from '../../../assets/topic.png';
 import dimen from '../../../utils/dimen';
-import {normalize, normalizeFontSizeByWidth} from '../../../utils/fonts';
+import {normalize} from '../../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `ImageBackground` is a new addition to the code, which is used for displaying the cover image. The removal of `normalizeFontSizeByWidth` from the imports seems fine as it's not being used in the updated code.
```diff
- import {Animated, StyleSheet, TouchableOpacity, View} from 'react-native';
+ import {Animated, ImageBackground, StyleSheet, TouchableOpacity, View} from 'react-native';
- import {normalize, normalizeFontSizeByWidth} from '../../../utils/fonts';
+ import {normalize} from '../../../utils/fonts';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->